### ISSUE TITLE
[Proposal] checking front-side-backward damage done percentage.

### ISSUE THREAD

**Issue by lustymoon** [neowutran/ShinraMeter#135]
Frankly, I have no idea how but one of KR user modified the meter and made this function. It is really helpful to enhance the performance, since backward stabbing is very important.

**Comment by neowutran** [comment]
Yup, this functionnality is planned for the next big release, but before the release, I need to do a huge refactoring of the code to increase the performance & to make new functionnality easier to dev. 
So no release before at least one or two week

**Comment by Gl0** [comment]
This function is done by me [here](https://github.com/neowutran/ShinraMeter/commit/ff1aceee97eb1fb7959538048e06096974aa1c3f), but without GUI and with some inconsistencies with real game logic. We need a bunch of ingame tests to determine how the game treats heading in different situations and made appropriate changes to meter logic.
